standalone/src/models/store/product-selection-setting/product-selection-setting-draft/presets/sample-data-b2b/default-product-selection.ts

```diff
@@ -1,8 +1,6 @@
 import { KeyReferenceDraft } from '../../../../../commons';
-import {
-  ProductSelectionDraft,
-  type TProductSelectionDraft,
-} from '../../../../../product-selection';
+import { ProductSelectionDraft } from '../../../../../product-selection';
+import type { TProductSelectionDraft } from '../../../../../product-selection/types';
```

Comment:
I wasn't sure if I should also export the types from the index file to avoid this change

Reply:
The product-selection index file should also export the types, yes.